<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1548</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCRtoODT GUI</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutCentral">
    <property name="spacing">
     <number>6</number>
    </property>


    <!-- Splitter: Левая и правая панель -->
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>

      <!-- Левая панель со списком файлов -->
      <widget class="QWidget" name="panelLeft">
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <item>
         <widget class="QLabel" name="lblFiles">
          <property name="text">
           <string>Список файлов</string>
          </property>
        </widget>
        </item>
        <item>
         <widget class="QListView" name="listFiles">
          <property name="minimumWidth">
           <number>220</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Правая панель: предпросмотр -->
      <widget class="QWidget" name="panelRight">
       <layout class="QVBoxLayout" name="verticalLayoutRight">
        <item>
         <widget class="QLabel" name="lblPreview">
          <property name="text">
           <string>Предпросмотр</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="viewPreview">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <!-- Кнопки Zoom -->
        <item>
         <layout class="QHBoxLayout" name="layoutZoomButtons">
          <item>
           <widget class="QPushButton" name="btnZoomIn">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoomOut">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoomFit">
            <property name="text">
             <string>Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoom100">
            <property name="text">
             <string>100%</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>

    <!-- Лог -->
    <item>
     <widget class="QTextEdit" name="textLog">
      <property name="minimumHeight">
       <number>100</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Лог работы...</string>
      </property>
     </widget>
    </item>
   </layout>

   <!-- распределяем вертикально вручную -->
   <zorder>splitterMain</zorder>
   <zorder>textLog</zorder>
  </widget>

  <!-- StatusBar -->
  <widget class="QStatusBar" name="statusBar">
   <widget class="QLabel" name="lblStatus">
    <property name="text">
     <string>Готов</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressTotal">
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>

  <!-- Toolbar -->
    <widget class="QToolBar" name="toolBar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
     <attribute name="toolBarArea">
      <enum>TopToolBarArea</enum>
     </attribute>
     <attribute name="toolBarBreak">
      <bool>false</bool>
     </attribute>

     <!-- Группа: ФАЙЛЫ -->
     <addaction name="actionOpen"/>
     <addaction name="actionClear"/>
     <addaction name="actionSettings"/>
     <addaction name="separator"/>

     <!-- Группа: OCR -->
     <addaction name="actionRun"/>
     <addaction name="actionStop"/>
     <addaction name="actionExport"/>
     <addaction name="separator"/>

     <!-- Группа: ПРОГРАММА -->
     <addaction name="actionAbout"/>
     <addaction name="separator"/>
     <addaction name="actionHelp"/>
    </widget>

  <!-- Actions -->
  <action name="actionOpen">
   <property name="text"><string>Открыть</string></property>
  </action>
  <action name="actionSettings">
   <property name="text"><string>Настройки</string></property>
  </action>
  <action name="actionRun">
   <property name="text"><string>Распознать</string></property>
  </action>
  <action name="actionStop">
   <property name="text"><string>Остановить</string></property>
  </action>
  <action name="actionExport">
   <property name="text"><string>Открыть ODT</string></property>
  </action>
  <action name="actionClear">
   <property name="text"><string>Очистить</string></property>
  </action>
  <action name="actionAbout">
   <property name="text"><string>О программе</string></property>
  </action>
  <action name="actionHelp">
   <property name="text"><string>Справка</string></property>
</action>


 </widget>
 <resources/>
 <connections/>
</ui>
